experimenter/databases/d3m_mtl.py

```diff
@@ -23,17 +23,17 @@ def __init__(self) -> None:
         self._post_url = D3M_MTL_DB_POST_URL
         # This env var allows code calling this class to be run during
         # unit tests without actually saving to the production DB.
-        self.should_save = config.SAVE_TO_D3M
+        self.should_save = config.D3MConfig().save_to_d3m
         # A reference to a low-level elasticsearch client. This can be
         # used to query the D3M DB, or this classe's `search` method
         # can be used, and is preferred, since its API is more straightforward.
         # This low-level client is the only way to accomplish
         # certain things though.
         self.es = Elasticsearch(hosts=[D3M_MTL_DB_GET_URL], timeout=30)
         # Our submitter name.
-        self._submitter = config.D3M_DB_SUBMITTER
+        self._submitter = config.D3MConfig().d3m_submitter
         # The secret verifying us as the submitter we say we are.
-        self._x_token = config.D3M_DB_TOKEN
+        self._x_token = config.D3MConfig().d3m_token
```

Comment:
These will also raise an error.

Reply:
Fixed issue. This was caused by a merge a little earlier I think, because I definitely attempted to fix it awhile ago.